src/lib/graphs.js

```diff
@@ -1,5 +1,6 @@
 "use strict";
 const Chart = require("chart.js");
+const rates = require("../augmint/rates.json");// every 50 days
```

Comment:
please don't make a dep on rates from graphs. see other comments how to overcome this